src/rulelib.cpp

```diff
@@ -90,23 +87,23 @@ rules_init(const char *infile, int *nrules,
 	 * the end.
 	 */
 	rule_cnt = add_default_rule != 0 ? 1 : 0;
-	while (getline(&line, &len, fi) != -1) {
+	while (m_getline(&line, &len, fi) != -1) {
```

Comment:
What's m_getline and does it have different behavior?

Reply:
Same with all the m_ functions

Reply:
So those were three functions that weren't present on Windows, as I think they're posix functions. In order to compile on Windows, I had to write my own implementations of them; see line 117 of utils.cpp.

Reply:
See my comments in utils.cpp